### ISSUE TITLE
future.utils.with_metaclass results in bogus no-value-for-parameter

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#353]
Originally reported by: **Alexey Kotlyarov (BitBucket: [koterpillar](http://bitbucket.org/koterpillar), GitHub: @koterpillar?)**

---

Pylint reports bogus warnings on the following code:

```
from future.utils import with_metaclass


class SomeClass(with_metaclass(type, object)):
    def __init__(self):
        super(SomeClass, self).__init__()


SomeClass()
```

```
E:  9, 0: No value for argument 'name' in constructor call (no-value-for-parameter)
E:  9, 0: No value for argument 'this_bases' in constructor call (no-value-for-parameter)
E:  9, 0: No value for argument 'd' in constructor call (no-value-for-parameter)
```

Versions of related packages:
- Pylint 1.3.1
- Future 0.13.1
- Python 2.7.8

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/353
